src/contactsbot/notesOperations.py

```diff
@@ -0,0 +1,21 @@
+from .decorators import input_error_catch, require_n_args, name_min_length
+from .NoteRecord import NoteRecord
+from .Notes import Notes
+from .helpers import print_with_color
+
+
+def note_add_handler(note: Notes):
+    """ note add hendler """
+    print_with_color('Please enter a note text : ', 'yellow')
+    note_text = input()
+    note.note_add(NoteRecord(note_text))
+    return "note added"
```

Comment:
Please capitalize it

Reply:
what are the capitalize, could you please provide example 
ASAP

Reply:
and [ ] - it is a feature! will be integrate in next commit (but now I will remove it)

Reply:
capitalize - is making first letter uppercase

Reply:
fixed